crates/proto/src/generated/block_producer.rs

```diff
@@ -16,6 +16,19 @@ pub struct SubmitProvenTransactionResponse {
     #[prost(fixed32, tag = "1")]
     pub block_height: u32,
 }
+#[derive(Clone, Copy, PartialEq, ::prost::Message)]
+pub struct SubmitProvenBatchResponse {
+    /// The node's current block height.
+    #[prost(fixed32, tag = "1")]
+    pub block_height: u32,
+}
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct ProvenBatch {
+    /// Encoded using \[winter_utils::Serializable\] implementation for
+    /// \[miden_objects::transaction::proven_tx::ProvenTransaction\].
+    #[prost(bytes = "vec", tag = "1")]
+    pub encoded: ::prost::alloc::vec::Vec<u8>,
```

Comment:
Do we want an enum here to do be able to have multiple formats / encodings eventually?

Reply:
Probably at some stage yes; but at the moment none of these are versioned so I've just kept with the standard.